src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -13,13 +13,18 @@
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import io.github.oblarg.oblog.annotations.Log;
 
 public class IndexSubsystem extends SubsystemBase {
```

Comment:
We should make this class implement `Loggable` to avoid the Oblog issues encountered earlier.